bundles/org.openhab.persistence.influxdb2/src/test/java/org/openhab/persistence/influxdb2/internal/ItemToStorePointCreatorTest.java

```diff
@@ -151,18 +169,18 @@ public static String itemName(String line) {
     }
 
     private static Map<String, String> parseFieldValueList(String fieldValueList) {
-        var fields = fieldValueList.split(",", -1);
-        var result = new HashMap<String, String>();
+        String[] fields = fieldValueList.split(",", -1);
+        Map<String, String> result = new HashMap<String, String>();
```

Comment:
You can write less code using the diamond operator:
 
```suggestion
        Map<String, String> result = new HashMap<>();
```